dev/feature/3424dfb.unknown-connection-sqlexecutequery/HelloWorldScript.yml

```diff
@@ -0,0 +1,14 @@
+---
+type: Script
+data:
+  type: "script"
+  name: "helloworld"
+  description: "Hello World Tutorial Script"
+  actions:
+  - number: 1
+    type: "fwk.dummy"
+    name: "HelloWorld"
+    description: "HelloWorld Action"
+    errorExpected: "N"
+    errorStop: "N"
+    parameters: []
```

Comment:
Are these files added to the repo as test or example scripts for documentation?

Reply:
These are the scripts to test the implementation